### DISCUSSION TITLE
Mountainduck consumes excessive CPU & Memory in MacOS

### DISCUSSION THREAD

**Discussion by minwoo-yu** [iterate-ch/cyberduck#16574] [Q&A]
While I'm using mountainduck on M2 Mac Mini (MacOS 15.1)
It consumes excessive CPU and RAM resources :(

These two figures are results when I connected at 2 storages via SFTP
<img width="819" alt="image" src="https://github.com/user-attachments/assets/7d2cbec7-7e30-48ee-8ad4-eacddf0f26f2">
<img width="838" alt="image" src="https://github.com/user-attachments/assets/acecaeb6-fda1-492a-8f90-abc89106970e">

**Comment by dkocher** [comment]
Discussed in #16431 and #14894.
